sql/20190424173430_create_clients_table.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE "clients" (
+    "id"                SERIAL  NOT NULL,
+    "first_name"        TEXT     NOT NULL,
+    "last_name"         TEXT     NOT NULL,
+    "birth_date"        TEXT    NOT NULL,
```

Comment:
date?

Reply:
hmm, may be we can do it, but for prototype we can use string here and in the future change it